webapp/src/css/inbox.less

```diff
@@ -93,178 +93,108 @@ body {
   margin: 0 auto;
 }
 
-.header {
-  display: none;
-  background-color: @top-header-color;
-  text-align: center;
-  padding-top: 5px;
+.app-root:not(.old-nav) mm-header {
+  height: 100vh;
+  width: @tab-nav-size;
+  position: fixed;
+  top: @filters-bar-desktop-height;
+  left: 0;
   z-index: 3;
+  overflow: hidden;
+}
+
+.app-root:not(.old-nav) .header {
   position: relative;
+  display: block;
+  z-index: 3;
+  background-color: @top-header-color;
```

Comment:
According to figma the background color should be `@medium-black`

Reply:
Ah good catch, I reused the variable for tab nav mobile. Just made the name more generic: @tab-navbar-color